dpc/src/transition/transition.rs

```diff
@@ -231,12 +225,12 @@ impl<N: Network> Transition<N> {
     ///
     #[inline]
     pub(crate) fn compute_transition_id(
-        serial_numbers: &[N::SerialNumber],
-        commitments: &[N::Commitment],
-        ciphertexts: &[RecordCiphertext<N>],
+        serial_numbers: &Vec<N::SerialNumber>,
+        commitments: &Vec<N::Commitment>,
+        _ciphertexts: &Vec<N::RecordCiphertext>,
```

Comment:
Note to self: I can remove the unused `_ciphertexts` from computing the transition ID, as it is now a part of the commitment.

Separate note to self: Add a check in `Transition::verify` (if possible) to check `ciphertext -> commitment` relationship.